src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/PosixSignalRegistrationTests.Unix.cs

```diff
@@ -14,19 +14,13 @@ public partial class PosixSignalRegistrationTests
     {
         public static IEnumerable<object[]> UninstallableSignals()
         {
-            if (PlatformDetection.IsNotMobile)
-            {
-                yield return new object[] { (PosixSignal)9 };
-            }
+            yield return new object[] { (PosixSignal)9 };
         }
 
         public static IEnumerable<object[]> SupportedSignals()
         {
-            if (PlatformDetection.IsNotMobile)
-            {
-                foreach (PosixSignal value in Enum.GetValues(typeof(PosixSignal)))
-                    yield return new object[] { value };
-            }
```

Comment:
FWIW, I hadn't removed these checks because it felt weird to me we were saying these values were supported when they actually weren't.  But, alas.

Reply:
Yeah it's a bit confusing but I think we have no other choice.